### DISCUSSION TITLE
Background Image

### DISCUSSION THREAD

**Discussion by meetmarvin** [kdevo/osprey-delight#24] [Q&A] [answered]
Hi,

What directory should I put the image and what to put in the config.yaml?

Thank you!

**Comment by kdevo** [comment, marked as answer]
Hi,

The preferred approach is to put the image in a high resolution under `assets/`.
Configuration key is `background` (under `Params`) in the config file, as shown in [the README table](https://github.com/kdevo/osprey-delight#images).

Are you using the `master` or the `dev` branch? 
The `v5` is currently in the making and might be released next month where the background configuration will be clarified a bit more. I've seen that the `background` is only partly documented in the exampleSite's config.yaml, so this definitely will be improved.

Hope it helps!
